Gemfile.lock

```diff
@@ -190,14 +190,14 @@ GEM
       request_store (~> 1.0)
     awesome_print (1.9.2)
     aws-eventstream (1.2.0)
-    aws-partitions (1.551.0)
-    aws-sdk-core (3.125.5)
+    aws-partitions (1.553.0)
+    aws-sdk-core (3.126.0)
```

Comment:
Just noting that a `bundle install` would have been better than a `bundle update`, as this is adding bumps to our production gems for a ticket that won't be manually tested. That said, the bumps are minor enough and limited enough that we can still release this w/o testing.

Reply:
Ah, yea, good catch.  Thanks for pointing this out.